gameci-1_intro.md

```diff
@@ -0,0 +1,146 @@
+Any changes outside of these folders would not cause our Unity project to change, so they can be ignored.
+For example, if we modify the README.md for our project, we wouldn't want the workflow to run, since it wouldn't actually be testing any relevant change to the Unity project.
+
+Runs on `release` will trigger whenever a release is created in the GitHub UI, and we can use `if: github.event.action == 'published'` to detect this scenario.
+
+The triggers for `workflow_dispatch` are the most interesting, since that's where we have setup the most control over the workflow.
+For example, if we wanted to create 2 Windows .exe's to download and run, we can simply pass `StandaloneWindows StandaloneWindows64` as the input.
+This would create both 32-bit and 64-bit Windows executables and upload them to GitHub, where they could then be downloaded for testing.
+
+Taking it a step further, we could input `release Steam` to run the entire production deployment pipeline for Steam.
+My Steam depots involve 4 artifacts: 1) Windows 32-bit, 2) Windows 64-bit, 3) Linux, and 4) Mac.
+Building and deploying these artifacts is split across multiple jobs, so it would now be good to get a high-level overview of all the jobs.
+
+## The Jobs
+
+Going into detail on each job would make this article too long.
+Furthermore, a reader who only wants to publish Android builds on Google Play would be interested in a different set of jobs than a reader who wants to publish PC games on Steam.
+Therefore, here is a quick overview of each job, with links to more info, if more info is desired.
+You may read these descriptions and then pick and choose to read only that which is relevant to you.
+
+### Test Code Quality
+I consider this job to be the most fundamental in any CI pipeline, as it is the one responsible for actually running your [unit tests](https://docs.unity3d.com/Manual/testing-editortestsrunner.html).
+I also set up [SonarQube](https://www.sonarqube.org/) quality checks and do some release management.
+See [GameCI 2: Testing](gameci-2_testing.html).
+
+### Build with Linux
+Builds the Android, iOS, StandaloneLinux64, and WebGL artifacts.
+See [GameCI 3: Build with Linux](gameci-3_linuxbuild.html).
+
+### Build with Mac
+Builds the StandaloneOSX artifact and signs it to generate `Card Game Simulator.app`.
+See [GameCI 5: Build and Deploy with Mac](gameci-5_mac.html).
+
+### Deploy to the App Store
+Takes the iOS artifact and uses fastlane to build and deploy to the App Store.
+See [GameCI 5: Build and Deploy with Mac](gameci-5_mac.html).
+
+### Deploy to the Google Play Store
+Takes the Android artifact and uses fastlane to deploy it to the Google Play Store.
+See [GameCI 4: Deploy with Linux](gameci-4_linuxdeploy.html).
+
+### Deploy to the Web via GitHub Pages
+Takes the WebGL artifact and deploys it via GitHub Pages.
+See [GameCI 4: Deploy with Linux](gameci-4_linuxdeploy.html).
```

Comment:
Consider changing the order of the steps themselves. Perhaps 4 could be build and deploy with Mac and 5 could be deploy with Linux (or also include the word build)